src/Shopsys/ShopBundle/Resources/views/Admin/Inline/Menu/menu.html.twig

```diff
@@ -0,0 +1,50 @@
+<ul class="js-toggle-menu navig navig--side">
+    <li class="js-toggle-menu-item navig__item">
+        <div class="box-dropdown">
+            <div
+                    class="box-dropdown__select"
+                    title="{{ 'Frontend'|trans }}"
+            >
+                {{ 'Go to Frontend'|trans }}
+                <i class="box-dropdown__select__arrow box-dropdown__select__arrow--up"></i>
+                <i class="box-dropdown__select__arrow box-dropdown__select__arrow--down"></i>
+            </div>
+            <ul class="js-toggle-menu-submenu box-dropdown__options">
+                {% for domainConfig in domainConfigs %}
+                    <li class="box-dropdown__options__item">
+                        <a href="{{ domainConfig.url }}" target="_blank" class="box-dropdown__options__item__link" title="{{ domainConfig.name }}">{{ domainConfig.url }}</a>
+                    </li>
+                {% endfor %}
+            </ul>
+        </div>
+    </li>
+
+    <li class="js-toggle-menu-item navig__item">
+        <div class="box-dropdown">
+            <div
+                    class="box-dropdown__select box-dropdown__select--with-icon"
+                    title="{{ 'Account'|trans }}"
+            >
+                <i class="svg svg-person-man"></i>
+                {{ app.user.realName }}
+                <i class="box-dropdown__select__arrow box-dropdown__select__arrow--up"></i>
+                <i class="box-dropdown__select__arrow box-dropdown__select__arrow--down"></i>
+            </div>
+            <ul class="js-toggle-menu-submenu box-dropdown__options">
+                
+                {% if isDefaultAdmin is same as (false) %}
```

Comment:
This should be also accessed by `app.user.defaultAdmin` without the need of passing a custom Twig variable.
```suggestion
                {% if not app.user.defaultAdmin %}
```

Reply:
Thank you, I did not know that fixed in 3251f445ac0a1bf7c718a11c043f940cc8cd89cc